packages/datatrak-web-server/src/app/createApp.ts

```diff
@@ -88,6 +90,7 @@ export async function createApp() {
     .get<TaskRequest>('tasks/:taskId', handleWith(TaskRoute))
     .get<SingleSurveyResponseRequest>('surveyResponse/:id', handleWith(SingleSurveyResponseRoute))
     .get<SurveyUsersRequest>('users/:surveyCode/:countryCode', handleWith(SurveyUsersRoute))
+    .get<PermissionGroupUsersRequest>('users/:countryCode', handleWith(PermissionGroupUsersRoute))
```

Comment:
I tossed up amending the `SurveyUsersRoute` to take a `permissionGroupName` parameter, or making this one more generic so that it could handle both cases, but I decided to create separate routes and move common logic to a util, because there would have been too much split logic to contend with and it would be hard to read later.

I also decided not to have `permissionGroupName` in the url params, but in the query instead, because permission group names have spaces

Reply:
The reasoning sounds good to me. One thing though is that I think that it's not great using the `permissionGroupName` in the api request. It's a shame there is no code so I guess we really should use the permissionGroupId. I guess the reason that you used the name is that it is more convenient as it is in the User question config? Perhaps we could transform the User question config to save the permissionGroupId?

Reply:
Good point, have made the name transform to be the id